### PR TITLE
pkg(com.taboola.scoop): add wallpaper carousel package

### PR DESCRIPTION
**Pull request by Unchanged4352**
Added com.taboola.scoop (Wallpaper Carousel)

### CODE REVIEW

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -38043,5 +38043,14 @@
...
+  },
+  "com.taboola.scoop": {
+    "list": "Oem",
+    "description": "Random wallpaper on lock screen (as i saw it personalizes it too), you need to open this app to make it work and when you swipe from right to left with your hand, it opens a menu on the right where random news appears.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
   }
+
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:38055]: Please remove this line, the pipeline is failing: https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/actions/runs/11765171736/job/32771550514?pr=714

File: resources/assets/uad_lists.json [hunk 2]
```diff
@@ -38043,5 +38043,14 @@
     "neededBy": [],
     "labels": [],
     "removal": "Expert"
+  },
+  "com.taboola.scoop": {
+    "list": "Oem",
+    "description": "Random wallpaper on lock screen (as i saw it personalizes it too), you need to open this app to make it work and when you swipe from right to left with your hand, it opens a menu on the right where random news appears.",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:38049]:
```suggestion
    "description": "Random wallpaper on lock screen (as i saw it personalizes it too). You need to open this app to make it work and when you swipe from right to left, it opens a menu on the right where random news appears.",
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

**Review by AnonymousWP — APPROVED** [review]
Thanks.🙂

**Comment by Unchanged4352** [comment]
> Thanks.🙂 

Np 🙂

**Pull request merged**